src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordGetHashCode.cs

```diff
@@ -83,6 +97,12 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 diagnostics.Add(ex.Diagnostic);
                 F.CloseMethod(F.ThrowNull());
             }
+
+            static void ensureEqualityComparerHelpers(SyntheticBoundNodeFactory F, ref MethodSymbol? equalityComparer_GetHashCode, ref MethodSymbol? equalityComparer_get_Default)
```

Comment:
Consider using `[NotNull]` on these parameters to avoid the need for `!` after calls to this method.

Reply:
> Consider using [NotNull] on these parameters to avoid the need for ! after calls to this method.

This is a local function, I believe we still don't allow attributes on them in our code base.

---
In reply to: [449217511](https://github.com/dotnet/roslyn/pull/45601#discussion_r449217511) [](ancestors = 449217511)